kong/plugins/acme/schema.lua

```diff
@@ -100,6 +100,10 @@ local schema = {
           default = 14,
         }, },
         { domains = typedefs.hosts },
+        { skip_domain_verify = {
```

Comment:
the option seems bit confusing as we are not really verifying a domain here, how about something like `allow_all_domains`